test/test_cuda_trace.py

```diff
@@ -19,18 +19,18 @@
 @torch.testing._internal.common_utils.markDynamoStrictTest
 class TestCudaTrace(TestCase):
     def setUp(self):
-        torch._C._activate_cuda_trace()
+        torch._C._activate_gpu_trace()
         self.mock = unittest.mock.MagicMock()
 
     def test_event_creation_callback(self):
-        cuda_trace.register_callback_for_cuda_event_creation(self.mock)
+        gpu_trace.register_callback_for_event_creation(self.mock)
```

Comment:
Since this test script is cuda-only. So you can set `cuda_trace` to `gpu_trace` like `cuda_trace = gpu_trace` to save loc.